packages/client-core/src/user/components/UserMenu/menus/AvatarSelectMenu.tsx

```diff
@@ -457,8 +460,8 @@ export const AvatarSelectMenu = (props: Props) => {
             type="button"
             className={styles.uploadBtn}
             onClick={uploadAvatar}
-            style={{ cursor: !fileSelected || !!error ? 'not-allowed' : 'pointer' }}
-            disabled={!fileSelected || !!error}
+            style={{ cursor: uploadButtonEnabled ? 'pointer' : 'not-allowed' }}
```

Comment:
Should the upload button be disabled once a file is selected? I can see someone mis-clicking and wanting to change the selection.

Reply:
the admin panel now uses the new logic and is working properly